images/win/Windows2016-Azure.json

```diff
@@ -458,6 +458,12 @@
                 "{{ template_dir }}/scripts/Installers/Install-AWS.ps1"
             ]
         },
+        {
+            "type": "powershell",
+            "scripts":[
+                "{{ template_dir }}/scripts/Installers/Install-AWS-SAM.ps1"
+            ]
```

Comment:
May be it makes sense to include installation of AWS-SAM to `Install-AWS.ps1`? The both are related to AWS so may be it makes sense to use the single file.

Reply:
I fact i tried this way - it is messy because of Install-AWS.ps1 includes 3 steps: installation, validation and documentation, also it uses choco